tests/unit_tests/test_gameconnection.py

```diff
@@ -147,16 +152,35 @@ def test_abort(game_connection, game, players):
 
         assert (game.id,) == await cursor.fetchone()
 
+
 async def test_handle_action_GameResult_victory_ends_sim(game, game_connection):
     game_connection.ConnectToHost = CoroMock()
     await game_connection.handle_action('GameResult', [0, 'victory'])
 
     assert game_connection.finished_sim
     assert game.check_sim_end.called
 
+
 async def test_handle_action_GameResult_draw_ends_sim(game, game_connection):
     game_connection.ConnectToHost = CoroMock()
     await game_connection.handle_action('GameResult', [0, 'draw'])
 
     assert game_connection.finished_sim
     assert game.check_sim_end.called
+
+
+async def test_handle_action_OperationComplete(game, game_connection):
+    pytest.skip("requires test data from faf-db:v63 which doesn't exist yet")
+
+    game.map_file_path = "maps/prothyon16.v0005.zip"
+    secondary = 1
+    time_taken = '09:08:07.654321'
+    await game_connection.handle_action('OperationComplete', ['1', secondary, time_taken])
+
+    import server.db as db
```

Comment:
don't we have fixture for db connection? If not, i wish that we would have it.

Reply:
We have a fixture for the db_pool in tests/conftest.py